### PR TITLE
GAWB-2104: Split "Public" and "My" Workspaces

### PR DESCRIPTION
**Pull request by MattPutnam**
Also did some work to use the new tabs everywhere, and make the 3 main pages look the same.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by rushtong** [comment]
When first navigating to the Method Repository, none of the sub-tabs were selected. The "All (XX)" should have been, but wasn't. Selecting any tab seems to reset the behavior correctly so that it will navigate to the right one on subsequent reloads. I'm guessing this has to do with local storage, but even after deleting my local storage, I could not replicate the original, unselected sub-tab condition.

**Comment by rushtong** [comment]
This needs a rebase, especially to pull in the shield-icon fix.

**Comment by rushtong** [comment]
For the jira ticket, please list each of the modified pages that need to be QA-ed in the QA section since the tab-style changes are more in-depth than the story describes.

**Comment by rushtong** [comment]
I'm confused by the counts that show in the workspaces tab when a filter is applied. In my example, I have 22 "My" workspaces and 23 "Public" workspaces. Applying a filter that reduces the number displayed to a single one looks strange. The "My" is empty, but shows a count of 22. The "Public" has 1, but shows a count of 23. Logically, I know that the counts are not reflected in the filter, but the visual effect is off. For example, if I leave that filter in place, leave, come back, and see the page again, it takes a split-second to realize what's happening.

Suggestion: I think the counts should update to reflect the filtered effect. Something like "My Workspaces (0 visible of 22 total)". I think this might also require PO feedback.

screen shot 2017-08-17 at 8 03 07 am

Edit: The behavior with filter vs facet is slightly different. When a facet is applied, the counts *do* update. Additionally, in combination, the count differences are even more jarring. Here is a screen with filter and facet selections with counts that only apply to the facet selection, but not the filter selection.

screen shot 2017-08-17 at 8 07 19 am

**Comment by rushtong** [comment]
In the condition where there are no rows in a table, the alignment of the background color block with the tabs is now more apparent compared to when the tabs didn't have a background. It now looks a bit jagged. Is it possible to align these? Not a big deal, but I think it would be cleaner if the right edge of the empty table aligned with the right edge of the tabs block above it. Screenshot shows old over the new.

screen shot 2017-08-17 at 8 17 09 am

File: src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs [hunk 1]
```diff
@@ -312,28 +312,33 @@
...
                         :name name
                         :snapshot-id snapshotId}
                     type (if (= entityType "Configuration") :method-config :method)]
                 (links/create-internal
-                 {:data-test-id (config/when-debug (str name "_" snapshotId))
-                  :href (if workspace-id "javascript:;" (nav/get-link type id))
-                  :onClick (when workspace-id
-                             #(swap! state assoc :type type :id id))}
-                 (style/render-name-id name snapshotId))))
+                  {:data-test-id (config/when-debug (str name "_" snapshotId))
```
Comment (rushtong) [inline src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs:327]: I think this is used for test automation - is that right?

**Review by rushtong — CHANGES_REQUESTED** [review]
Overall, looks really good! 
* I have one request about counts that I think should be addressed in some way.
* My request for table alignment is minor - don't let that hold things up.

File: src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs (hunk 1 above)
Comment (ahaessly) [reply src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs:327]: yes, it is

**Review by ahaessly — APPROVED** [review]
👍

**Comment by ahaessly** [comment]
saw a minor issue - if the browser is resized to something narrow the table on the launch analysis screen extends past the boundary
screen shot 2017-08-18 at 2 59 17 pm

**Review by rushtong — APPROVED** [review]

**Pull request merged**